src/preemptive/lwt_preemptive.ml

```diff
@@ -207,20 +207,20 @@ let detach f args =
   in
   Lwt.finalize
     (fun () ->
-      (* Send the id and the task to the worker: *)
+       (* Send the id and the task to the worker: *)
        CELL.set worker.task_cell (id, task);
        waiter)
     (fun () ->
-      if worker.reuse then
-        (* Put back the worker to the pool: *)
-        add_worker worker
-      else begin
-        decr threads_count;
-        (* Or wait for the thread to terminates, to free its associated
-           resources: *)
-        Thread.join worker.thread
-      end;
-      Lwt.return_unit)
+       if worker.reuse then
```

Comment:
IMO ocp-indent gets this kind of thing wrong. It shouldn't cause an extra space if you have to add a `(` to a function (or any other expression). Is it easy to fix in the ocp-indent settings? If not, it's fine – no need to go through all these files. The PR does much more good than harm, even with this nit.